pages/common/prettier.md

```diff
@@ -0,0 +1,24 @@
+# prettier
+
+> An opinionated code formatter for JavaScript, JSON, CSS, YAML, and more.
+> More information: <https://prettier.io/>.
+
+- Format a file and print the result to stdout:
+
+`prettier {{path/to/file}}`
+
+- Check if a specific file has been formatted:
+
+`prettier --check {{path/to/file}}`
+
+- Run with a specific configuration file:
+
+`prettier --config {{path/to/config_file}} {{path/to/file}}`
+
+- Format a file in place:
+
+`prettier --write {{path/to/file}}`
+
+- Format all JavaScript files in the current directory recursively:
+
+`prettier --write '*.js'`
```

Comment:
This is kind of pointless, and it isn't recursive through directories.

This command is just a longer version of the main command, and anyone should be able to identify the wildcard (*). 

It does have a use, but not on this page since it is common throughout *nix using *

Reply:
(Just to make it easier for OP) 

```
```

Reply:
Yeah, the command is not recursive, could be useful an example like `only format js files` because you're saying this format different files like YAML.

Could be useful an example with prettier settings in the CLI like:
```
prettier --single-quote --trailing-comma {{none}} --write {{path/to/directory}}
```

Reply:
_If we decide to keep the command, it still needs to be changed since it implies that only JavaScript files can be done._

I could see the usefulness of that command suggested, however I would either remove it entirely or make it recursive by change it to */*/* or something similar ( I couldn't see a -R or -r flag)

Reply:
To make it recursive you'd need to replace `'*.js'` with `"$(find . -type f -iname '*.js')"`, but that's dangerously close to demonstrating other commands' functionality. I recommend just updating the exampled description.